internal/pkg/service/buffer/condition/checker.go

```diff
@@ -226,26 +215,8 @@ func (c *Checker) watchTokens(ctx context.Context, wg *sync.WaitGroup) (errCh <-
 		}).
 		StartMirroring(wg)
 
-	// Invalidate cache on distribution cache.
-	// See WithFilter above, ownership is changed on distribution change, so cache must be re-generated.
-	wg.Add(1)
-	go func() {
-		defer wg.Done()
```

Comment:
Toto tu bolo zbytocne 3x, teraz je to iba raz a invaliduje to vsetky 3 mirrors/caches.